src/cow-react/modules/limitOrders/updaters/QuoteUpdater/hooks/useFetchMarketPrice/useHandleResponse.ts

```diff
@@ -36,11 +37,16 @@ export function useHandleResponse() {
         const adjustedBuy = JSBI.multiply(JSBI.BigInt(buyAmount), JSBI.BigInt(10 ** getDecimals(inputCurrency)))
         const adjustedSell = JSBI.multiply(JSBI.BigInt(sellAmount), JSBI.BigInt(10 ** getDecimals(outputCurrency)))
 
-        // Create executionRate fraction
+        // Create execution rate fraction
         const executionRate = new Fraction(adjustedBuy, adjustedSell)
 
+        // Apply -0.1% to execution rate
+        const change = String(0.1 / 100)
+        const toSubtract = executionRate.multiply(toFraction(change))
```

Comment:
We should have a util for this, I think is common to apply a percentage. 

BTW in case its helpful, this is what we have for something very similar
https://github.com/cowprotocol/cowswap/blob/f458608b5e059726e020f5ececf822bac9c433e5/src/custom/state/swap/TradeGp.ts#L52